testproject/Packages/packages-lock.json

```diff
@@ -69,7 +69,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.mathematics": {
-      "version": "1.1.0",
+      "version": "1.2.1",
```

Comment:
same question if it was an intentional update

Reply:
Fixed.